dss/util/checkout.py

```diff
@@ -0,0 +1,127 @@
+import io
+import uuid
+from enum import Enum, auto
+from logging import getLogger
+
+import chainedawslambda
+from chainedawslambda import aws
+from chainedawslambda.s3copyclient import S3ParallelCopySupervisorTask
+
+from cloud_blobstore import BlobNotFoundError, BlobStoreUnknownError
+from cloud_blobstore.s3 import S3BlobStore
+from dss import chained_lambda_clients, DSSException, Config
+from dss.util.aws import get_s3_chunk_size
+from dss.util.bundles import get_bundle, get_bundle_from_bucket
+
+for client_name, client_class in chained_lambda_clients():
+    chainedawslambda.aws.add_client(client_name, client_class)
+
+log = getLogger()
+blobstore = S3BlobStore()
+
+
+class BundleFileMeta:
+    NAME = "name"
+    UUID = "uuid"
+    VERSION = "version"
+    CONTENT_TYPE = "content-type"
+    INDEXED = "indexed"
+    CRC32C = "crc32c"
+    S3_ETAG = "s3_etag"
+    SHA1 = "sha1"
+    SHA256 = "sha256"
+
+class ValidationEnum(Enum):
+    NO_SRC_BUNDLE_FOUND = auto(),
+    WRONG_DST_BUCKET = auto(),
+    WRONG_PERMISSIONS_DST_BUCKET = auto(),
+    WRONG_BUNDLE_KEY = auto(),
+    PASSED = auto()
+
+def parallel_copy(
+        source_bucket: str, source_key: str,
```

Comment:
I would prefer that you don't use this newline+indent style. Functions should look like this:
```python
def fn(a: T, b: T, ...,
       c: T, d: T):
    pass

Reply:
fixed